src/styles/components/footer/footer.module.scss

```diff
@@ -0,0 +1,124 @@
+.footer {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-maroon);
+}
+
+.body {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 36px 146px;
+    color: var(--color-gold);
+}
+
+.pages {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+
+    font-weight: bold;
+    font-size: 1rem;
+    gap: 22px;
+    list-style: none;
+}
+
+.link {
+    text-decoration: none;
+    color: var(--color-gold);
+}
+
+.contact {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    p {
+        display: flex;
+        justify-content: center;
+        text-align: center;
+        line-height: 1.5;
+    }
+    h3 {
+        display: flex;
+        justify-content: center;
+    }
+
+    width: 25%;
+    font-size: 1rem;
+    gap: 5px;
+
+}
+
+.icons {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.circle_frame {
```

Comment:
I noticed the only difference between `circle_frame` and `circle_frame_facebook` happens to be the font size. We can instead create one `circle_frame` class and create an additional `facebook` class that changes the font_size. This helps to avoid repetition. Then, in the HTML you give the `circle_frame` class to all icons and only add the additonal `facebook` class to the facebook icon (you can play around with the names ofc, just an example)

Reply:
ahhhh that makes so much sense. I knew there had to be a better way of doing it